### ISSUE TITLE
Failed to get IHolographicCameraRenderingParameters from main camera for updating rendering parameter.

### ISSUE THREAD

**Issue by CDiaz-MS** [microsoft/MixedRealityToolkit-Unity#6807]
## Describe the bug

This error is logged when a hand enters the frame during remoting with HoloLens2.
```
Failed to get IHolographicCameraRenderingParameters from main camera for updating rendering parameter.
```

![image](https://user-images.githubusercontent.com/53493796/70265971-622a8e00-1750-11ea-8cf1-75458592dcc0.png)

## To reproduce

1. Install remoting app on HoloLens 2 device
2. Import latest unity foundation and examples packages, open hand interaction examples scene
3. Window > XR > Holographic Emulation, enter ip address displayed through device 
4. https://docs.microsoft.com/en-us/windows/mixed-reality/unity-play-mode

## Your Setup 

- Unity Version 2018.4.13f1
- MRTK Version v2.2

## Target Platform 
- HoloLens 2 Remoting

**Comment by davidkline-ms** [comment]
i cannot reproduce this with the latest Windows Mixed Reality XR package (1.0.15) on Unity 2018.4.13f1

**Comment by Troy-Ferrell** [comment]
@davidkline-ms / @keveleigh 
I can reproduce this on 2019.2.17f1

Looks like maybe Unity bug underneath?
https://docs.microsoft.com/en-us/uwp/api/windows.graphics.holographic.holographiccamerarenderingparameters

**Comment by keveleigh** [comment]
Spent a bit debugging this. Looks like a Unity issue, possibly caused / exacerbated by any perf cost of hand visualization / remoting. Unfortunately doesn't have an obvious fix that I can see, but I'll keep my eye on it.

**Comment by nooxouille** [comment]
Hi there! Issue reproduced on Unity **2018.4.18f1** (LTS) + latest MRTK version

**Comment by keveleigh** [comment]
We've identified a potential fix with Unity. I'll update this thread when I get an expected version number.

**Comment by ceitel** [comment]
Also reproduced with Unity 2019.3.3f1, no MRTK installed, WMR 4.2.1 package installed

**Comment by matware** [comment]
Seeing this in 2019.3.6f1 in clean scene with MR toolkit 2.3.0 (no hand mesh) . Also not seeing anything on device, it connects, screen goes black and nothing, but I see a room mesh/camera movement in the scene, so there is data going back to Unity.

Using the new XR plugin management system and the Windows XR Remoting plugin I do get video output, but no hand meshes etc.

Does anybody have Holographic Remoting working reliably with Unity day to day? If so, how?

**Comment by keveleigh** [comment]
Looking like the fix for this should ship in Unity 2019.3.7 and 2018.4.19.

**Comment by nooxouille** [comment]
> 
> 
> Looking like the fix for this should ship in Unity 2019.3.7 and 2018.4.19.

Nice! Many thanks ;)

**Comment by ceitel** [comment]
2019.3.7 is now available! (as well as 2018.4.20)

**Comment by matware** [comment]
Yay, i finally checked and can also confirm that it work.

**Comment by keveleigh** [comment]
Closing as fixed in Unity 2019.3.7 and 2018.4.19. Please comment if you're still seeing it in those versions!
